swarm/storage/mru/resource.go

```diff
@@ -38,12 +39,11 @@ import (
 )
 
 const (
-	signatureLength         = 65
```

Comment:
This moved to `updaterequest.go`. At the end of the refactor in PR #704, it will end up where signatures actually take place.